### PR TITLE
improve: layout

### PR DESCRIPTION
**Pull request by leeoocca**

### CODE REVIEW

**Comment by itislu** [comment]
I don't like that once there is one long issue title, the table has to be horizontally scrolled to see all columns.
image

Also, the maximum issue title length (50 characters) should at least on desktop be fully in one line.
image

**Comment by leeoocca** [comment]
> Was the narrower layout on mobile intentional?

Ah yes, probably forgot to remove `p-4` on the single pages.

File: src/app/routes/_app.issues.$id.tsx [hunk 1]
```diff
@@ -387,8 +386,7 @@ export default function IssueDetail() {
     }
 
     return (
-        <div>
-            <NavBar login={session.login} role={session.role} />
+        <Fragment>
             <div className='md:flex md:justify-center'>
                 <div className='md:w-3/5 p-4'>
```
Comment (leeoocca) [inline src/app/routes/_app.issues.$id.tsx:391]: Most likely here

File: src/app/routes/_app.polls.$id.tsx [hunk 2]
```diff
@@ -221,8 +213,7 @@ export default function IssueDetail() {
     }
 
     return (
-        <div>
-            <NavBar login={session.login} role={session.role} />
+        <Fragment>
             <div className='md:flex md:justify-center'>
                 <div className='md:w-3/5 p-4'>
```
Comment (leeoocca) [inline src/app/routes/_app.polls.$id.tsx:218]: And here

**Comment by leeoocca** [comment]
I had missed some things... now it should be better. Many things are now simplified and more coherent (mostly in the code) without major changes in the final rendering apart from a couple of fixes mentioned in the last comment.

**Comment by itislu** [comment]
Yup now the layout is not much narrower than the live version, so my previous comments are solved :)
I see a lot of small improvements now with the spacing between elements!
Bc it's a big code change I'll let @ifaoji have the last say on this PR.

**Comment by leeoocca** [comment]
Ok just fixed the conflicts

**Review by ifaoji — APPROVED** [review]

**Comment by ifaoji** [comment]
plz fix conflicts

**Review by ifaoji — APPROVED** [review]

**Pull request merged**